schutzbot/sonarqube.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+export SONAR_SCANNER_OPTS="-Djavax.net.ssl.trustStore=schutzbot/RH-IT-Root-CA.keystore -Djavax.net.ssl.trustStorePassword=$KEYSTORE_PASS"
+sudo dnf install -y unzip
+curl "https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-$SONAR_SCANNER_CLI_VERSION-linux.zip" -o sonar-scanner-cli.zip
```

Comment:
I've defined it as a CI variable so that we can easily change it without the need for a PR in case we need it, but I suppose I can put at least a default value here.